charts/reports/templates/_helpers.tpl

```diff
@@ -31,13 +31,20 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Selector labels
+*/}}
+{{- define "reports.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "reports.name" . }}
```

Comment:
Don't forget the unit test for this change (check the `metadata.labels` for non regression ;) )